src/main/java/com/kaleido/fetch/service/ActivityService.java

```diff
@@ -113,40 +114,73 @@ public Activity getActivity(Long id) {
     }
 
     public ResponseEntity<String> saveActivityDraft(PlateMap plateMap) {
-    	
-    	log.info("Platemap data is ", plateMap);
-    	String plateMapURI = cabinetURI + "plate-maps";
-    	
-    	return (ResponseEntity<String>) cabinetClient.cabinetPlatemap(plateMapURI, plateMap, HttpMethod.PUT, String.class);
+        log.info("Platemap data is ", plateMap);
+        String plateMapURI = cabinetURI + "plate-maps";
+        
+        return (ResponseEntity<String>) cabinetClient.cabinetPlatemap(plateMapURI, plateMap, HttpMethod.PUT, String.class);
     }
     
     public ResponseEntity<String> saveCompletedActivity(PlateMap plateMap) {
-    	
-    	log.info("Platemap data is ", plateMap);
-    	String plateMapURI = cabinetURI + "plate-maps";
-    	
-    	return (ResponseEntity<String>) cabinetClient.cabinetPlatemap(plateMapURI, plateMap, HttpMethod.PUT, String.class);
+        log.info("Platemap data is ", plateMap);
```

Comment:
```suggestion
        log.debug("Platemap data is ", plateMap);
```
The plateMap object is going to be big